app/_containers/Main/index.tsx

```diff
@@ -1,17 +1,36 @@
 import cx from "clsx";
 
+import { notFound } from "next/navigation";
 import SectionAboveTheFold from "./containers/SectionAboveTheFold";
 import styles from "./index.module.scss";
+import SectionArticles from "./containers/SectionArticles";
+import { httpGet$GetPosts } from "@/modules/commands/GetPosts/fetcher";
+import { intentionallyIgnoreError } from "@/modules/error/intentionallyIgnoreError";
 
 type Props = {
   className?: string;
   style?: React.CSSProperties;
 };
 
-export default function Main({ className, style }: Props) {
+const ARTICLE_CATEGORY_ID = 3;
+
+export default async function Main({ className, style }: Props) {
+  const data = await httpGet$GetPosts(`/wp-json/wp/v2/posts`, {
+    categories: ARTICLE_CATEGORY_ID,
+  }).catch(intentionallyIgnoreError);
+  if (!data) {
+    notFound();
+  }
+  // Fetch latest post
+  const sortedPosts = data.sort((a: { date: string }, b: { date: string }) => {return new Date(b.date).getTime() - new Date(a.date).getTime();
+        });
+  const three_latest = sortedPosts.slice(0, 3);
```

Comment:
I think the API should be able to filter and sort. Please research how to do it.